sunpy/map/mapbase.py

```diff
@@ -1414,6 +1410,44 @@ def superpixel(self, dimensions: u.pixel, offset: u.pixel=(0, 0)*u.pixel, func=n
         return new_map
 
 # #### Visualization #### #
+    @property
+    def cmap(self):
+        """
+        Colormap used for plotting the map.
+        """
+        return self._cmap
+
+    @cmap.setter
+    def cmap_set(self, cmap):
+        self._cmap = cmap
+
+    @property
+    def norm(self):
+        """
+        Normalisation to use for plotting the map.
+        """
+        return self._norm
+
+    @norm.setter
+    def norm_set(self, norm):
+        self._norm = norm
+
+    @property
+    def plot_settings(self):
+        """
+        Plot settings given to imshow.
+        """
+        return {'cmap': self.cmap,
+                'norm': self.norm,
+                'interpolation': 'nearest',
+                'origin': 'lower'}
+
+    @plot_settings.setter
+    def plot_settings_set(self, plot_settings):
+        if 'cmap' in plot_settings:
+            self.cmap = plot_settings['cmap']
+        if 'norm' in plot_settings:
+            self.norm = plot_settings['norm']
```

Comment:
You still need to set `plot_settings` here, as it's a way of setting arbitrary keyword argument to imshow.